tests/integ/test_temp_table_cleanup.py

```diff
@@ -253,9 +253,10 @@ def test_session_close(db_parameters, caplog):
 def test_auto_clean_up_temp_table_enabled_parameter(db_parameters, session, caplog):
     warning_dict.clear()
     with caplog.at_level(logging.WARNING):
-        session.auto_clean_up_temp_table_enabled = False
-    assert session.auto_clean_up_temp_table_enabled is False
+        session.auto_clean_up_temp_table_enabled = True
+    assert session.auto_clean_up_temp_table_enabled is True
     assert "auto_clean_up_temp_table_enabled is experimental" in caplog.text
+    session.auto_clean_up_temp_table_enabled = False
```

Comment:
do we not raise warning only when we enable the param?

Reply:
The warning is only raised when setting to a non-default value.